rust/notus/src/notus.rs

```diff
@@ -7,42 +7,55 @@ use std::collections::HashMap;
 use models::{NotusResults, VulnerablePackage};
 
 use crate::{
-    advisory::{Advisories, PackageAdvisories},
     error::Error,
-    loader::{AdvisoriesLoader, FeedStamp},
+    loader::{FeedStamp, ProductLoader},
     packages::Package,
+    vts::{Product, VulnerabilityTests},
 };
 
+/// Notus is a tool to check for vulnerabilities on you system based on the installed packages.
+/// In order for it to work other tools must be used to collect these packages from your system.
+/// Additionally a loader must be given, which loads the list of vulnerable packages, including
+/// their versions. It is recommended to use a safe loader like the `HashsumFileLoader`, which
+/// loads a product file based of a Hashsum file. Look into the `HashsumFileLoader` for more
+/// information.
+///
+/// Notus will cache loaded products to increase speed for repeted scans of the same systems and
+/// automatically updates the cache, when the acording file changes.
```

Comment:
rs-typo CI test fails because of this 
```suggestion
/// automatically updates the cache, when the according file changes.
```